### ISSUE TITLE
[Task] <In the Documentation Contribution Guidelines, add the Documentation Writing Guidelines>

### ISSUE THREAD

**Issue by yuluo-yx** [apache/hertzbeat#2648]
### Description

Nowadays, there is no standardization of document writing, we can use [Docs]https://github.com/ruanyf/document-style-guide] as a guide to standardize the use of documents.

@tomsun28 @zhangshenghang @zqr10159, how do you think?

**Comment by tomsun28** [comment]
hi we already has the markdown lint check https://github.com/apache/hertzbeat/blob/master/.markdownlint-cli2.jsonc , it is also a standardization of document writing.

**Comment by yuluo-yx** [comment]
> hi we already has the markdown lint check https://github.com/apache/hertzbeat/blob/master/.markdownlint-cli2.jsonc , it is also a standardization of document writing.

I think they are different. The format check of md is aimed at the fixed md writing format. And what is in the link is the content. I think they can compare it to ci checking the style of the composition. The function of this link is how to write a more vivid composition.

**Comment by zhangshenghang** [comment]
I looked at the above document, it can be recommended as a best practice, it cannot be used as a requirement to constrain everyone, I think format check still uses markdownlint-cli2.jsonc, You can improve the configuration rules of markdownlint documents to make the documents more standardized.

**Comment by yuluo-yx** [comment]
> I looked at the above document, it can be recommended as a best practice, it cannot be used as a requirement to constrain everyone, I think format check still uses markdownlint-cli2.jsonc, You can improve the configuration rules of markdownlint documents to make the documents more standardized.

yep, That's what I mean, use him as a best practice guide, not a mandatory requirement, and refer to it when writing.
